exercises/storage/storage03.cairo

```diff
@@ -10,16 +10,22 @@ end
 
 @external
 func toggle{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}():
-    # TODO
-    # Implement toggle external
+    let (b) = bool.read()
```

Comment:
Hum isn't it the solution?

Reply:
Yeah I messed up. I dunno how I ended up pushing this one too